code/modules/multiz/turf.dm

```diff
@@ -169,8 +169,13 @@ see multiz/movement.dm for some info.
 			ReplaceWithLattice()
 		return
 
-	if (istype(C, /obj/item/stack/material/steel))
+	if (istype(C, /obj/item/stack/material))
+		var/obj/item/stack/material/M = C
+		if (!istype(M.material, /material/steel))
```

Comment:
It's better to use `get_material` instead direct var check.

Reply:
sure, will do